project.py

```diff
@@ -193,6 +299,12 @@ def video_search(self, video):
 
 def main(inputFile):
     board = BoardSearcher()
+    board.numberOfSlides = 0
+    board.frameCounter = 0
+    # how many matches we have to find to not save
+    board.similarity = 0.70
```

Comment:
@Foxiy based on the comment above this looks like we need to find `0.7` matches. I am not sure but I believe that matches are discrete objects and you cannot really have a 0.7 of a match. Either this value is wrong or the comment above this line is.

It might also be that I am completely wrong so please do let me know which one it is.